src/interface/src/app/forget-password/forget-password.component.scss

```diff
@@ -0,0 +1,90 @@
+.forget-password-root {
```

Comment:
you can avoid creating this extra element and just use `:host` to style the component

Reply:
It wasn't working for me when I switched this to `:host`. The flex display is not applied.

Reply:
This is OK by me, thanks for checking!